docs/docs/references/RELEASE.md

```diff
@@ -0,0 +1,684 @@
+# Release notes
+
+*Contains description of Nebari releases.*
+
+<!-- Note:
+The RELEASE.md file at the root of the Nebari codebase is the source of truth for all release notes.
+If you want to update the release notes, open a PR against nebari-dev/nebari.
+This file is copied to nebari-dev/nebari-docs using a GitHub Action. -->
+
+---
+
+## Upcoming Release
+
+### Feature changes and enhancements
+
+## Release v0.4.5 - October 14, 2022
```

Comment:
Are we going to re-release under the nebari name? Or do we need to make a note here that this is under the qhub name?